binding.gyp

```diff
@@ -13,12 +13,18 @@
         'VCCLCompilerTool': {
           'AdditionalOptions': [
             '/Qspectre',
-            '/guard:cf'
+            '/guard:cf',
+            '/ZH:SHA_256'
+            '/w34244',
+            '/w34267',
+            '/sdl'
           ]
         },
         'VCLinkerTool': {
           'AdditionalOptions': [
-            '/guard:cf'
+            '/guard:cf',
+            '/CEFCOMPAT',
+            '/DYNAMICBASE'

```

Comment:
Can you adjust the flags here similar to https://github.com/microsoft/vscode-windows-ca-certs/pull/7#pullrequestreview-1296666210